src/test/scala/uk/gov/hmrc/play/views/layouts/FooterSpec.scala

```diff
@@ -19,18 +19,15 @@ package uk.gov.hmrc.play.views.layouts
 import org.scalatest.{Matchers, WordSpec}
 import play.api.test.Helpers.{contentAsString, defaultAwaitTimeout}
 import play.twirl.api.Html
-import uk.gov.hmrc.play.test.NoStartedPlayApp
 import uk.gov.hmrc.play.views.html.layouts.Footer
 import uk.gov.hmrc.play.views.layouts.test.TestConfigs._
 
-class FooterSpec extends WordSpec with Matchers with NoStartedPlayApp {
+class FooterSpec extends WordSpec with Matchers {
   "footer" should {
 
     val footer = new Footer(testAssetsConfig)
 
     "be renderable without a started Play application" in {
```

Comment:
I think this test needs looking at - I think, although I could be wrong, that this test would be for the case when not using DI because there is no started application, i.e. this tests the static templates. This may need another look. @matthewmascord have you looked at this play-ui test before?

Reply:
yeah this one seemed a bit odd to me, it's not testing DI or static templates because it wires up it's own version on line 29/28 - it seems like it's showing that the footer isn't using anything not passed by DI that requires there be a running play application - I thought maybe to prove that the component could be used in a service where play wasn't being used

Reply:
looks like we could start a version of play within the test suite and pass

```
play.allowGlobalApplication=false
`
``

which will throw an exception if the component is accessing the global application

though not starting the application feels like it would be enough to me to cover the test description without that